src/app/garfile/cancel/index.njk

```diff
@@ -25,17 +25,22 @@
   {% endif %}
 
   <h1 class="govuk-heading-l">
-    Are you sure you want to cancel this GAR ?
+    {{__('heading_cancel_gar')}}
   </h1>
   <p class="govuk-body-m">
-    You will need to ensure that a new GAR is submitted if this flight takes place.
+    {{__('cancel_gar_caption')}}
   </p>
   <br>
   <form action="/garfile/cancel" method="post">
-    <input class="govuk-button" type="submit" value="Cancel GAR">
+    <button id="cancel-gar" class="govuk-button" type="submit">{{__('title_cancel_gar')}}</button>
```

Comment:
WCAG suggests button is a little better than input with type="submit", so will need to double check if this is the case